src/components/Breadcrumb/BreadcrumbView.js

```diff
@@ -13,6 +13,7 @@ const CustomBreadcrumb = ({ classes, data }) => (
               to={current.to}
               onClick={current.onClick}
               key={current.to}
+              id={`${index + 1}_breadcrumb`}
```

Comment:
@iksheth Looks like Branch is outdated with the integration, can you take a look.

Reply:
Sure